html/lib/db/pdo.php

```diff
@@ -0,0 +1,17 @@
+<?php 
+
+function execute(string $sql, array $args) : PDOStatement
+{
+	static $pdo;
+	$pdo ??= new PDO('odbc:dsn=soma');
```

Comment:
This does kind of depend on your odbc setup, interested to know if this works out the box